typescript-examples/data-example/src/App.tsx

```diff
@@ -32,7 +32,7 @@ export const App: FC = () => {
 
     const initMap = async () => {
       const map = await createMap({
-        apiKey: "<your-api-key>",
+        apiKey: "fsq3xmchm2MR7alFdHFopcPvLY3QvY2A1CnyIYEqxJ1Mdy8=",
```

Comment:
Can these keys be abused?

Reply:
I wrote more about it [here](https://foursquare.atlassian.net/browse/UN-7028).